demo/scripts/controls/ContentModelEditorMainPane.tsx

```diff
@@ -198,6 +201,12 @@ class ContentModelEditorMainPane extends MainPaneBase<ContentModelMainPaneState>
             this.sampleEntityPlugin,
         ];
 
+        for (let i = 0; i < plugins.length; i++) {
```

Comment:
you can remove it from demo\scripts\controls\getToggleablePlugins.ts then no need to remove from here